<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesConfiguration</class>
 <widget class="QDialog" name="NotesConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="colorsBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="bgColorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bgColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fgColorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Foreground</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="fgColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="showWindowFrame">
     <property name="text">
      <string>Show window frame</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="displayFont">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="showNotesOnStartup">
     <property name="text">
      <string>Show notes on startup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="changeFont">
     <property name="text">
      <string>Default font</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
